staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -104,6 +113,12 @@ func init() {
 	prometheus.MustRegister(requestLatenciesSummary)
 	prometheus.MustRegister(responseSizes)
 	prometheus.MustRegister(DroppedRequests)
+	prometheus.MustRegister(currentInflightRequests)
+}
+
+func UpdateInflightRequestMetrics(nonmutating, mutating int) {
```

Comment:
This metric seems like it belongs in the filter package, not here.  We shouldn't have to register metrics in this package.  This is a weird coupling.